packages/@vue/cli/bin/vue.js

```diff
@@ -261,7 +258,14 @@ if (!process.argv.slice(2).length) {
 function suggestCommands (unknownCommand) {
   const availableCommands = program.commands.map(cmd => cmd._name)
 
-  const suggestion = didYouMean(unknownCommand, availableCommands)
+  let suggestion
+
+  availableCommands.forEach(cmd => {
+    if (leven(cmd, unknownCommand) < 3) {
```

Comment:
```suggestion
    if (leven(cmd, unknownCommand) < 3 && leven(cmd, unknownCommand) < leven(suggestion || '', unknwonCommand)) {
```

(Of course, you can also memoize the result of `leven(suggestion || '', unknwonCommand)` to a variable to make the code cleaner. But that's the basic idea.